### PR TITLE
Test rspec

### PR DESCRIPTION
**Pull request by prolajumokeoni**
####  In  this project:
- Setup  new branch and installed RSpec 
- Built test for all public methods in Players and Game classes 
- Wrapped all specs for each method in a describe block
- Tested for negative and positive test scenarios
##### Implemented changes requested by previous TSE

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      subject.count
+      subject.add_board(4)
+      subject.count
+      subject.add_board(3)
+      subject.count
+      subject.add_board(8)
+      subject.count
+      subject.add_board(9)
+      expect(subject.won?).to be_a(Array)
+      expect(subject.won?).to eql([2, 5, 8])
```
Comment (george-shammar) [inline spec/game_spec.rb:54]: - [ ] You should have one "it" block for each expected behaviour.

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: :construction: 

**Priscilla, this is a strong and impressive first submission**. :+1: 
- Your test cases all passed. :clap: 
- You covered all the public methods in your TicTacToe. :clap: 
- Your test cases are easy to understand. 

Overall, you've done a great job.

There's a little bit of changes required in order to make this submission really really good. Please see my inline comment as well. :point_down: :point_down: 

- [ ] You should have one "it" block for each expected behaviour. 
- [ ] Please include negative test scenarios. For example, there is a spec for correct usage of the method [e.g. it "creates object if all required attributes are passed"]
and also there is spec for the wrong usage of a method [e.g. it "does not create the object if required attributes are missing"].

**Please make the required changes and you should be good to go.** :hammer: :hammer_and_pick: 

If you require help, do not hesitate to drop your message here and hit me up on slack.

**Submit another CRR when you're ready.**

**Cheers and happy coding.** :smile:

**Comment by prolajumokeoni** [comment]
Hi @george-shammar, can you explain more on including negative test scenarios and wrong usage of examples.

**Comment by george-shammar** [comment]
Hi @prolajumokeoni , I'll use one of your test cases as an example here. 

**Positive test case:**

it 'returns 0 on the board if ' do
      subject.add_board(2)
      expect(subject.board).to eql([1, 'O', 3, 4, 5, 6, 7, 8, 9])
 end

**Negative test case:**
 it 'does not return to on the board if ' do
      subject.add_board(2)
      expect(subject.board).not_to eql([1, 2, 3, 4, 5, 6, 7, 8, 9])
 end

Hope this helps. If it is still not clear, feel free to reach out on slack. Perhaps, we could schedule a short call.

**Comment by prolajumokeoni** [comment]
> Hi @prolajumokeoni , I'll use one of your test cases as an example here.
> 
> **Positive test case:**
> 
> it 'returns 0 on the board if ' do
> subject.add_board(2)
> expect(subject.board).to eql([1, 'O', 3, 4, 5, 6, 7, 8, 9])
> end
> 
> **Negative test case:**
> 
> it 'does not return on the board if ' do
> subject.add_board(2)
> expect(subject.board).not_to eql([1, 2, 3, 4, 5, 6, 7, 8, 9])
> end
> 
> Hope this helps. If it is still not clear, feel free to reach out on slack. Perhaps, we could schedule a short call.

Thanks @george-shammar, I get it now.

**Review by elisha2kyakpo1 — APPROVED** [review]
# Status: Approved✔🎉💪
Congratulations! I'm happy that you have managed to make all the changes requested by the previous reviewer, continue working hard.

Cheers🎉

Elisha

**Pull request closed without merging**